tests/emails/0.yml

```diff
@@ -0,0 +1,16 @@
+subject: 'zaÅ¼Ã³ÅÄ gÄÅlÄ jaÅºÅ i zrÃ³b prÃ³bne zadanie %%tod'
+from:
+  - name: Pawel
+    email: pawel@test.com
+to:
+  - name: ''
+    email: dan@test.com
+cc: []
+bcc: []
+sender: []
+plainbody: |
+            This is a nonesense test-email to check Plancake Mail to Inbox feature. 
+            All Polish diacritics included ;-) tag also included for additional tests.
+            It should look like in the picture attached.
+            Best regards
+            Pawel
```

Comment:
I didn't give them newlines since that line is included in the multiline field, making the test fail.

Fixed it with another syntax in yaml.